notebooks/120-tf2-object-detection-to-openvino/120-tf2-object-detection-to-openvino.ipynb

```diff
@@ -0,0 +1,579 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Convert a TensorFlow 2 Object Detection Model to OpenVINO™\n",
+    "\n",
+    "[TensorFlow](https://www.tensorflow.org/), often referred to as TF, is an open source framework for machine learning. \n",
+    "\n",
+    "This tutorial shows how to convert a TensorFlow 2 [Faster R-CNN with Resnet-50 V1](https://tfhub.dev/tensorflow/faster_rcnn/resnet50_v1_640x640/1) object detection model to OpenVINO [Intermediate Representation](https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_IR_and_opsets.html) (OpenVINO IR) format, using [Model Optimizer](https://docs.openvino.ai/2023.0/openvino_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html). After creating the OpenVINO IR, load the model in [OpenVINO Runtime](https://docs.openvino.ai/nightly/openvino_docs_OV_UG_OV_Runtime_User_Guide.html) and do inference with a sample image."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Prerequisites\n",
+    "\n",
+    "Install required packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q \\\n",
```

Comment:
@ryanloney pip install command was fixed. Could you please try it on Windows 11 one more time?

Reply:
@yatarkan still not working for me: 
<img width="605" alt="image" src="https://github.com/openvinotoolkit/openvino_notebooks/assets/15709723/2a77ee97-38b2-444c-9f42-1c8bf3720f3f">

Reply:
@ryanloney I have tried to change Jupyter kernel spec to use `python3` instead of local env, I hope it will resolve the issue. Can I ask you to check it again?

Reply:
@yatarkan problem in quotas, please replace ' to " (' wrongly interpreted in windows)

Reply:
Quotes fixed, thanks